docs/source/futures.rst

```diff
@@ -12,6 +12,15 @@ These features depend on the second generation task scheduler found in
 `dask.distributed <https://distributed.dask.org/en/latest>`_ (which,
 despite its name, runs very well on a single machine).
 
+.. raw:: html
+
+   <iframe width="560"
+           height="315"
+           src="https://www.youtube.com/embed/4gLaEbchZW0"
```

Comment:
@mrocklin The video is great, good content on futures, but it was a little harder to read the content because of quality (360p). Is it available in better quality?